packages/frontend/src/app/repository/tests-list/tests-list.component.html

```diff
@@ -15,49 +15,62 @@
 -->
 
 <div id="tests-container">
+    
+    <mat-accordion multi>
+        <mat-expansion-panel *ngFor="let test of renderedElements" 
```

Comment:
I love that we're able to use these off the shelf components, and get such delightful behavior.